--- sglang/srt/configs/qwen2_5_vl_config.py
+++ sglang/srt/configs/qwen2_5_vl_config.py
@@ -1002,5 +1002,9 @@
         return BatchFeature(data=data, tensor_type=return_tensors)
 
 
-AutoImageProcessor.register(Qwen2_5_VLConfig, None, Qwen2_5_VLImageProcessor, None)
-AutoProcessor.register(Qwen2_5_VLConfig, Qwen2_5_VLProcessor)
+AutoImageProcessor.register(
+    Qwen2_5_VLConfig, None, Qwen2_5_VLImageProcessor, None, exist_ok=True
+)
+AutoProcessor.register(
+    Qwen2_5_VLConfig, Qwen2_5_VLProcessor, exist_ok=True
+)
